problems/make_it_molecular/verify.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+var exec = require('child_process').exec;
+var fs = require('fs');
+var path = require('path');
+var filename = "01-item.mustache";
+
+//check the directory exists
+//check the file exists
+//verify the contents of the file
+
+findFile();
+
+function findFile() {
+  if (process.cwd().match("molecules")) {
+    check(process.cwd())
+  } else {
+    check(path.join(process.cwd(), "/molecules/"))
+  }
+
+  function check(userspath) {
+    fs.readdir(userspath, function(err, files) {
+      if (err) return console.log(err);
```

Comment:
What I am alluding to is 

if (err) return console.log(err);

Is valid js there is a lot of style guides that talk about the right or wrong style:
https://github.com/RisingStack/node-style-guide#blocks